internal/status/registry.go

```diff
@@ -201,7 +201,8 @@ func (r *Registry) List(ctx context.Context) ([]Registration, error) {
 		// the server is dead, we consider the deployment dead, and we garbage
 		// collect its registration.
 		if r.dead(ctx, reg) {
-			r.Unregister(ctx, reg.DeploymentId)
+			// TODO: log this error instead of ignoring it
+			r.Unregister(ctx, reg.DeploymentId) //nolint:errcheck
```

Comment:
I'm not sure how important it is to log this error. I believe this operation is some form of cleanup when a previous deployment is terminated forcefully. I suspect it would be helpful to log the error if the deployment can't be cleaned up.

How would I get access to the logger here? Is there currently any way to get a logger from `ctx` ? I did not find one.

Reply:
Maybe we should just write the error to `os.Stderr`? I don't think there's a nice way to get a logger here. What do you think?

Reply:
I looked at the callers to this function, and I think that should work well. I see it being called primarily from CLI commands, and from one HTTP handler that is started by a CLI command. I've made that change in the latest commit!

Storing and retrieving a `Logger` from `ctx` has some nice properties, but maybe is not appropriate in this case. Either way I imagine it's not a priority right now, so maybe something to explore in the future.